lib/srv/db/mongodb/protocol/opmsg.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 Gravitational, Inc.
+Copyright 2022 Gravitational, Inc.
```

Comment:
Shouldn't that be `2021-2022`?
```suggestion
Copyright 2021-2022 Gravitational, Inc.
```